webapp/src/css/enketo/medic.less

```diff
@@ -60,14 +60,18 @@
   .btn-primary {
     background-color: @button-color;
   }
-  .btn-icon-only {
+  .btn-icon-only:not(.draw-widget__undo) {
     padding: 0;
     width: inherit;
   }
-  .icon {
+  .icon:not(.draw-widget__undo .icon-undo) {
     position: relative;
     top: 1px;
   }
+  // Hide download button for draw widget
```

Comment:
Hiding the download button (lets you download a copy of your drawing before submitting the form). It did not work for some reason on Android (maybe a permissions problem?) and seemed like extra clutter to include on the page unless we have a use-case where we need it....